pkgs/development/tools/build-managers/cask/default.nix

```diff
@@ -1,10 +1,10 @@
-{ emacsPackagesNg, writeScriptBin }:
+{ emacsPackagesNg, writeScriptBin, lib }:
 let
 
   emacs = emacsPackagesNg.emacsWithPackages (epkgs: [ epkgs.cask-package-toolset ]);
   cpt = emacsPackagesNg.cask-package-toolset;
 
-in writeScriptBin "cask" ''
+in writeScriptBin "cask-${lib.getVersion emacs}" ''
```

Comment:
This would rename the script.

Reply:
Ok good catch! I'll just leave that out then.